### PR TITLE
Updated README, Moved local camera view to picture-in-picture

### PR DESCRIPTION
**Pull request by jefflinwood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,78 @@ Use this project to get started with Twilio Video's Android SDK by [following al
...
+
+You'll see how to set up key classes like TwilioAccessManager, ConversationsClient, LocalMedia, Conversation, and CameraCapturer. All of these Twilio classes have related listeners with implementations in ConversationActivity.java.
+
+You can also see how Android's runtime permissions are implemented for access to the camera and the microphone on Android devices running version 6.0 (Marshmallow) or higher inside the onCreate() method in the ConversationActivity class. It's important to note that you shouldn't initialize the Twilio client until the user grants access to the camera and microphone.
+
+## Prerequisites
+
+This project is a native Android application written in Java. You will need an up-to-date version of Android Studio (at the time of this writing, 2.1.0), to build the application. You will also need version 23.0.2 of the Android build tools, which you can download through Android Studio if you do not have them installed.
+
+You should use an Android device for testing - the minimum supported Android version is 4.0.3.
```
Comment (aaalaniz) [inline README.md]: This is not a hard requirement anymore. The latest conversations SDK supports the x86 emulators (32 and 64 bit)

**Comment by dreamerns** [comment]
Awesome job on PIP mode. 👍

**Comment by aaalaniz** [comment]
other than my one comment 👍

File: README.md (hunk 1 above)
Comment (jefflinwood) [inline README.md]: Glad to hear it! I'll make a note of that (and try it out).

**Comment by aaalaniz** [comment]
👍

**Pull request merged**